src/txmempool.h

```diff
@@ -751,8 +751,12 @@ class CTxMemPool
  */
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
+public:
+    void AddPotentialTransaction(const CTransactionRef& ptx) { package_tx.emplace(ptx->GetHash(), ptx); }
```

Comment:
As far as I see, in the current code code paths, the max number of elements in `package_tx` is two.
But maybe there should be a hard limit at the level just in case?